BigBaseV2/src/core/globals.hpp

```diff
@@ -64,6 +64,8 @@ namespace big
 			};
 
 			bool godmode = false;
+			bool invisibility = false;
+			bool clean_player = false;
```

Comment:
Set these alphabetically, to follow the same coding style as the entire project.